<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyTwoDialog</class>
 <widget class="QDialog" name="StudyTwoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudyTwoTool</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="leftWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="ballWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="simulateLabel">
             <property name="styleSheet">
              <string notr="true">     font: bold 28px;</string>
             </property>
             <property name="text">
              <string>Simulate Ball 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="simulateBall2Button">
             <property name="styleSheet">
              <string notr="true"> QPushButton {
     background-color: #ecf0f1;
     border-style: outset;
     border-width: 2px;
     border-color: #2c3e50;
     font: bold 28px;
     padding: 6px;
 }
 QPushButton:hover {
     background-color: #bdc3c7;
     border-style: inset;
 }
 QPushButton:pressed {
     background-color: #95a5a6;
     border-style: inset;
 }</string>
             </property>
             <property name="text">
              <string>Simulate Next</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="showChooseDialogButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"> QPushButton {
     background-color: #ecf0f1;
     border-style: outset;
     border-width: 2px;
     border-color: #2c3e50;
     font: bold 28px;
     padding: 6px;
 }
 QPushButton:hover {
     background-color: #bdc3c7;
     border-style: inset;
 }
 QPushButton:pressed {
     background-color: #95a5a6;
     border-style: inset;
 }</string>
             </property>
             <property name="text">
              <string>Choose</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="chooseWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="ballOneSelectedButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"> QPushButton {
     background-color: #ecf0f1;
     border-style: outset;
     border-width: 2px;
     border-color: #2c3e50;
     font: bold 28px;
     min-width: 10em;
     padding: 6px;
 }
 QPushButton:hover {
     background-color: #bdc3c7;
     border-style: inset;
 }
 QPushButton:pressed {
     background-color: #95a5a6;
     border-style: inset;
 }</string>
             </property>
             <property name="text">
              <string>Ball 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ballTwoSelectedButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"> QPushButton {
     background-color: #ecf0f1;
     border-style: outset;
     border-width: 2px;
     border-color: #2c3e50;
     font: bold 28px;
     min-width: 10em;
     padding: 6px;
 }
 QPushButton:hover {
     background-color: #bdc3c7;
     border-style: inset;
 }
 QPushButton:pressed {
     background-color: #95a5a6;
     border-style: inset;
 }</string>
             </property>
             <property name="text">
              <string>Ball 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#2c3e50;</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"> QGroupBox {
     border: 1px solid #34495e;
	

     margin-top: 10px; 
 }

 QGroupBox::title {
     subcontrol-origin:margin;
     subcontrol-position: top left;
     padding: -8 5px;
	color:white;
 }</string>
          </property>
          <property name="title">
           <string>Results</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Survey</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QWidget" name="widget_10" native="true">
                <layout class="QFormLayout" name="formLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QWidget" name="widget_9" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="styleSheet">
                       <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
                      </property>
                      <property name="text">
                       <string>Feedback Type</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QWidget" name="widget_8" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="styleSheet">
                       <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
                      </property>
                      <property name="text">
                       <string>Trial</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QWidget" name="widget_7" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_9">
                      <property name="styleSheet">
                       <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
                      </property>
                      <property name="text">
                       <string>Condition</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QWidget" name="widget_6" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_7">
                      <property name="styleSheet">
                       <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
                      </property>
                      <property name="text">
                       <string>User</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="surveyFeedbackLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel{
	background: #34495e;
	padding: 10px;
	color: #f1c40f;
}</string>
                   </property>
                   <property name="text">
                    <string>Value</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="surveyTrialLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel{
	background: #34495e;
	padding: 10px;
	color: #f1c40f;
}</string>
                   </property>
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="surveyConditionLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel{
	background: #34495e;
	padding: 10px;
	color: #f1c40f;
}</string>
                   </property>
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLabel" name="surveyUserLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel{
	background: #34495e;
	padding: 10px;
	color: #f1c40f;
}</string>
                   </property>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="styleSheet">
              <string notr="true"> QGroupBox {
     border: 1px solid #34495e;

     margin-top: 10px; 
 }

 QGroupBox::title {
     subcontrol-origin:margin;
     subcontrol-position: top left;
     padding: -8 5px;
	color:white;
 }</string>
             </property>
             <property name="title">
              <string>Ball Properties</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="styleSheet">
                 <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
                </property>
                <property name="text">
                 <string>Radius Ball 1: (cm)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="conditionBallRadius">
                <property name="styleSheet">
                 <string notr="true"> QLineEdit {
     border: 0px solid gray;
     padding: 10px;
     background: #34495e;
     selection-background-color: #34495e;
	color: #f1c40f;
 }
</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
                </property>
                <property name="text">
                 <string>Radius Ball 2:(cm)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="conditionBall2Radius">
                <property name="styleSheet">
                 <string notr="true"> QLineEdit {
     border: 0px solid gray;
     padding: 10px;
     background: #34495e;
     selection-background-color: #34495e;
	color: #f1c40f;
 }
</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="styleSheet">
              <string notr="true"> QGroupBox {
     border: 1px solid #34495e;

     margin-top: 10px; 
 }

 QGroupBox::title {
     subcontrol-origin:margin;
     subcontrol-position: top left;
     padding: -8 5px;
	color:white;
 }</string>
             </property>
             <property name="title">
              <string>User</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_11">
                <property name="styleSheet">
                 <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
                </property>
                <property name="text">
                 <string>Radius: (cm)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="userFingerRadiusLineEdit">
                <property name="styleSheet">
                 <string notr="true"> QLineEdit {
     border: 0px solid gray;
     padding: 10px;
     background: #34495e;
     selection-background-color: #34495e;
	color: #f1c40f;
 }
</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>On</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>Off</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
